azure-batch/src/main/java/com/microsoft/azure/batch/CreateTasksTerminatedException.java

```diff
@@ -25,7 +25,7 @@
      * @param pendingList The list of pending tasks
      */
     public CreateTasksTerminatedException(final String message, List<TaskAddResult> failureTasks, List<TaskAddParameter> pendingList) {
-        super(message);
```

Comment:
@xingwu1 It's now recommended that AutoRest generated exceptions carry response and response body (if any). But for now, the quick and dirty way is to set them both to null. Is this okay with you?